api/app/events/api.py

```diff
@@ -419,15 +381,24 @@ def get(self):
         args = self.req_parser.parse_args()
 
         user_id = g.current_user['id']
+        language = args['language']
+        if language is None or len(language) > 2:
```

Comment:
So I think Cyprus was doing something strange here. Looking at my logs in the console I saw this. It was sending a `language` parameter with `en-US` and that test was failing hence why I'm defaulting to `en` if the length of the language parameter is greater than 2 (i.e. the length allowed by the ISO 639-1 standard). 
`172.17.0.5 - - [10/Aug/2020 11:58:34] "GET /api/v1/events?cache_bust=1&language=en-US HTTP/1.1" 401 -`

Reply:
This is an issue with the front-end (or rather a TODO) - I added a plugin that automatically sets the language to the browser default, but I still need to add some logic to filter it to the list of organisation supported languages